src/common/LogFormatter.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+#include "pch.h"
+#include "spdlog/pattern_formatter.h"
+#include "utils.h"
+
+/**
+ * Custom formatter for spdlog. Same as default but using the getTimeStamp() function
+ */
+class LogFormatter : public spdlog::formatter
```

Comment:
```suggestion
class FileLogFormatter : public spdlog::formatter
```

This is only used for files, right?

Reply:
I'm kinda bad with names lol